### DISCUSSION TITLE
Get Device Information

### DISCUSSION THREAD

**Discussion by deqw** [astrandb/miele#206] [Q&A] [answered]
Thanks for this nice integration!
Is there a way to get to the device information for own use (model, firmware etc), like shown on the Integration Device page?

**Comment by astrandb** [comment, marked as answer]
Do you mean in a sensor or as an attribute? Serial number is shown as an attribute to the status sensor.

**Reply by deqw** [reply]
I like to add the information to my dashboard. I saw the serial number in the status, maybe add it there since this will be most static information. But will I then be able to use the dashboard editor with it?

**Reply by astrandb** [reply]
It is possible. 
Please make a feature request at https://github.com/astrandb/miele/issues/new/choose

**Reply by deqw** [reply]
Will do. Thanks!
